CHANGELOG.md

```diff
@@ -15,6 +15,7 @@
 ### Removed
 
 ### Bug Fixes
+- Fixed log formatting issue that occurred while loading the configuration for Ansible-based operators. ([#2246](https://github.com/operator-framework/operator-sdk/pull/2246))
```

Comment:
It needs to be in the unreleased section.

Reply:
It looks like it already is.

Reply:
solved.